internal/runners/clean/run.go

```diff
@@ -31,3 +35,29 @@ func undoPrepare(cfg configurable) error {
 
 	return aggErr
 }
+
+func removeEnvPaths(cfg configurable) error {
+	isAdmin, err := osutils.IsWindowsAdmin()
+	if err != nil {
+		return errs.Wrap(err, "Could not determine if running as Windows administrator")
+	}
+
+	// remove shell file additions
+	s := subshell.New(cfg)
+	if err := s.CleanUserEnv(cfg, sscommon.InstallID, isAdmin); err != nil {
+		return errs.Wrap(err, "Failed to State Tool installation PATH")
+	}
+	// Default and deployed projects will stop working, so we return them from the PATH as well
```

Comment:
I was not 100% sure whether we should also remove default and deploy directory PATHs here, but as the executors they contain will not work without the State Tool, IMO this should be the correct behavior.

Without removing the entries, the user may get errors like "Could not find executable `state`" when calling `python` or `perl` which I believe is very confusing to the average user.